python/ray/tune/BUILD

```diff
@@ -230,7 +230,7 @@ py_test(
     size = "medium",
     srcs = ["tests/test_warnings.py"],
     deps = [":tune_lib"],
-    tags = ["team:ml", "exclusive", "no_new_storage"],
+    tags = ["team:ml", "exclusive", "new_storage"],
```

Comment:
Keeping the `new_storage` tag and the 2 CI runners because `doctest` is not migrated yet.